student/src/main/java/tds/student/services/PrintService.java

```diff
@@ -248,16 +278,30 @@ public boolean printItemBraille (TestOpportunity testOpp, ItemResponse responseT
         _tdsLogger.rendererWarn (error, "printItemBraille");
         return false;
       }
+
+      ITSAttachment mainBrailleAttachment = brailleAttachments.get(0);
+
+      // the subType in the Item Metadata attachmentList is the only way to determine if there is a Braille Transcript file
+      //  it will follow the convention of being named with an _transcript suffix according to the specification
+      boolean isTranscript = brailleAttachments.size() == 2 && brailleAttachments.get(1).getSubType().endsWith("_transcript");
```

Comment:
Possibly extract this call out to a private static method that could do the check?  So the conditional lives in a single place.